job-server/justfile

```diff
@@ -0,0 +1,86 @@
+# Load .env files by default
+set dotenv-load := true
+
+# check that everything local job-server is configured
+check:
+    #!/bin/bash
+    set -euo pipefail
+
+    error() { 
+        echo "${1:-}"
+        echo "You probably to run just job-runner/configure"
+        exit 1
+    }
+
+    test -f .env.jobserver || error
+
+    . .env.jobserver
+    test "SOCIAL_AUTH_GITHUB_KEY" != "test" || error "SOCIAL_AUTH_GITHUB_KEY not set properly"
```

Comment:
Can we make this check that it's set non-empty, as well as not "test"? In my first failed attempt, the `.env.jobserver` file was created and had `SOCIAL_AUTH_GITHUB_SECRET=` in it, so it didn't ask me for the bw password again.

Reply:
Ah, yes, that's a bit brittle, will do